README.md

```diff
@@ -1 +1,108 @@
-# kuadrant-operator
+# Kuadrant Operator
+
+[![License](https://img.shields.io/badge/license-Apache--2.0-blue.svg)](http://www.apache.org/licenses/LICENSE-2.0)
+
+## Overview
+
+The Operator to install and manage the lifecycle of the [Kuadrant](https://github.com/Kuadrant/) components deployments.
+
+### Provided APIs
+
+The kuadrant control plane owns the following [Custom Resource Definitions, CRDs](https://kubernetes.io/docs/tasks/extend-kubernetes/custom-resources/custom-resource-definitions/):
+
+| CRD                                                                                                                                                 | Description |
+|-----------------------------------------------------------------------------------------------------------------------------------------------------| --- |
+| RateLimitPolicy CRD [\[doc\]](https://github.com/Kuadrant/kuadrant-controller/blob/main/doc/rate-limiting.md) [[reference]](https://github.com/Kuadrant/kuadrant-controller/blob/main/doc/ratelimitpolicy-reference.md) | Enable access control on workloads based on HTTP rate limiting |
+| [AuthPolicy CRD](https://github.com/Kuadrant/kuadrant-controller/blob/main/apis/apim/v1alpha1/authpolicy_types.go)                                                                                            | Enable AuthN and AuthZ based access control on workloads |
+
+Additionally, kuadrant provides the following CRDs
+
+| CRD | Owner | Description |
+| --- | --- | --- |
+| [Kuadrant CRD](https://github.com/Kuadrant/kuadrant-operator/blob/main/api/v1beta1/kuadrant_types.go) | [Kuadrant Operator](https://github.com/Kuadrant/kuadrant-operator) | Represents an instance of kuadrant |
+| [Limitador CRD](doc/ratelimitpolicy-reference.md) | [Limitador Operator](https://github.com/Kuadrant/limitador-operator) | Represents an instance of Limitador |
+| [Authorino CRD](https://github.com/Kuadrant/authorino-operator#the-authorino-custom-resource-definition-crd) | [Authorino Operator](https://github.com/Kuadrant/authorino-operator) | Represents an instance of Authorino |
+| [AuthConfig CRD](https://github.com/Kuadrant/authorino/blob/main/docs/architecture.md#the-authorino-authconfig-custom-resource-definition-crd) | [Authorino](https://github.com/Kuadrant/authorino) | The desired authN and authZ protection for a service |
+Examples:
+
+#### Kuadrant CR
+```yaml
+---
+apiVersion: kuadrant.kuadrant.io/v1beta1
+kind: Kuadrant
+metadata:
+  name: kuadrant-sample
+spec: {}
+```
+
+#### RateLimitPolicy CR
+```yaml
+---
```

Comment:
I think I would link to the sample, as this could change and become out of date?

Reply:
+1

Reply:
Makes sense, although these 2 last example CRs belong to the _kuadrant-controller_ repo, I could link them nevertheless

Reply:
or should we just show the operator's CRDs only?

Reply:
Even if they belong to the [kuadrant-controller](https://github.com/Kuadrant/kuadrant-controller) repo, they are provided via this operator when installing kuadrant. So I think it is ok to add here. 

The CRD table might have an additional column with links to examples (hosted in the [kuadrant-controller](https://github.com/Kuadrant/kuadrant-controller)).

Anyway, I would add an explicit link to the doc site. Something like:

```
# Documentation

Docs can be found on the [Kuadrant website](https://kuadrant.io/).
```

Reply:
>I think I would link to the sample, as this could change and become out of date?

[asciidoc ftw!](https://docs.asciidoctor.org/asciidoc/latest/directives/include/)